src/Modules/UngradedModules.jl

```diff
@@ -3267,13 +3392,27 @@ function gens(F::SubQuo, G::FreeMod)
 end
 rels(F::FreeMod) = elem_type(F)[]
 
+@doc Markdown.doc"""
+    ambient_representatives_generators(M::SubQuo)
+
+Return elements of the ambient free module of `M` which represent the generators of `M`.
+"""
+ambient_representatives_generators(M::SubQuo) = gens(M, ambient_free_module(M))
```

Comment:
Huh, I wasn't aware that `gens` can take two arguments?! What does that mean? OK, I just looked it up in the documentation:
>  gens(F::SubQuo, G::FreeMod)
>
>  Return the generators of F as elements of G where G is the embedding free module.

That doesn't sound very natural to me in a variant of `gens`. In particular, i think the only allowed way to call this `gens is in exactly the way `ambient_representatives_generators` does, yes?

So perhaps this `gens` should be removed. And perhaps instead there should be a `lift` method, so that `lift(x)` for `x` an element of a SubQuo returns the corresponding element in the ambient space? 

@fieker @thofma thoughts?

Reply:
Our idea on our todo list was the following: as soon as this is merged, @AlexD97 will replace gens(F::SubQuo, G::FreeMod) throughout by ambient_representatives_generators(M::SubQuo). I agree that this term is rather long ...